content/contributing/opencue/getting-started-windows.md

```diff
@@ -0,0 +1,186 @@
+---
+title: "Development Environment setup on Windows"
+linkTitle: "Development Environment setup on Windows"
+date: 2020-01-29
+weight: 2
+description: >
+  A guide to setting up an end-to-end development environment on Windows.
+---
+
+<!---DO NOT MODIFY THIS LINE--->
+
+This is a guide to setting up a development environment on Windows that will allow 
+you to run the entire OpenCue system from end-to-end (Cuebot, Cuesubmit, CueGUI, RQD)
+and make changes to any part of it.
+
+## Prerequisites
+
+First, clone the [OpenCue git repository](https://github.com/AcademySoftwareFoundation/OpenCue)
+to your machine. You will then need to download & install three core dependencies:
+
+- [Postgres](https://www.postgresql.org/download/windows/)
+- [Python 3.x](https://www.python.org/downloads/) 
+- [Java SE JDK](https://www.oracle.com/technetwork/java/javase/downloads/index.html)
+
+It is also useful to have IDEs installed. JetBrains has free versions of both:
+
+- [PyCharm](https://www.jetbrains.com/pycharm/)
+- [IntelliJ IDEA](https://www.jetbrains.com/idea/)
+
+## Database setup
+
+After installing Postgres you will need to create a database and user for OpenCue
+(specifically CueBot) to use.
+
+1. **Start pgAdmin**: Open pgAdmin from the Start menu. This will add an icon to the system tray. 
+   Right-click the pgAdmin icon and select “New pgAdmin window…”, which will open 
```

Comment:
For UI interactions, our style is to bold the text, so 

Click **Save**

instead of 

Click "Save"

Don't worry about fixing this PR though, I can update it after we merge your contributions.

Reply:
This is easy to fix; a lot cleaner afterwards as well 🙂